src/connectedvmware/azext_connectedvmware/tests/latest/test_connectedvmware_scenario.py

```diff
@@ -0,0 +1,111 @@
+            'vm_name': 'azcli-test-virtual-machine'
+        })
+
+        # Validate the show command output with vcenter name.
+        self.cmd('az connectedvmware vcenter show_command -g {rg} --name {vc_name}', checks=[
+            self.check('name', '{vc_name}'),
+        ])
+
+        # Count the vcenter resources in this resource group with list command
+        count = len(self.cmd('az connectedvmware vcenter list -g {rg}').get_output_in_json())
+        # vcenter count list should report 1
+        self.assertEqual(count, 1, 'vcenter resource count expected to be 1')
+
+        # Create resource-pool resource.
+        self.cmd('az connectedvmware resource-pool create -g {rg} -l {loc} --custom-location {cus_loc} --vcenter {vc_name} --mo-ref-id {rp_morefid} --name {rp_name}')
+
+        # Validate the show command output with resource-pool name.
+        self.cmd('az connectedvmware resource-pool show_command -g {rg} --name {rp_name}', checks=[
+            self.check('name', '{rp_name}'),
+        ])
+
+        # List the resource-pool resources in this resource group.
+        resource_list = self.cmd('az connectedvmware resource-pool list -g {rg}').get_output_in_json()
+        # At this point there should be 1 resource-pool resource.
+        assert len(resource_list) >= 1
+
+        # Create virtual-network resource.
+        self.cmd('az connectedvmware virtual-network create -g {rg} -l {loc} --custom-location {cus_loc} --vcenter {vc_name} --mo-ref-id {vnet_morefid} --name {vnet_name}')
+
+        # Validate the show command output with virtual-network name.
+        self.cmd('az connectedvmware virtual-network show_command -g {rg} --name {vnet_name}', checks=[
+            self.check('name', '{vnet_name}'),
+        ])
+
+        # List the virtual-network resources in this resource group.
+        resource_list = self.cmd('az connectedvmware virtual-network list -g {rg}').get_output_in_json()
+        # At this point there should be 1 virtual-network resource.
+        assert len(resource_list) >= 1
+
+        # Create vm-template resource.
+        self.cmd('az connectedvmware vm-template create -g {rg} -l {loc} --custom-location {cus_loc} --vcenter {vc_name} --mo-ref-id {vmtpl_morefid} --name {vmtpl_name}')
+
+        # Validate the show command output with vm-template name.
+        self.cmd('az connectedvmware vm-template show_command -g {rg} --name {vmtpl_name}', checks=[
+            self.check('name', '{vmtpl_name}'),
+        ])
+
+        # List the vm-template resources in this resource group.
+        resource_list = self.cmd('az connectedvmware vm-template list -g {rg}').get_output_in_json()
+        # At this point there should be 1 vm-template resource.
+        assert len(resource_list) >= 1
+
+        # Validate the show command output with inventory-item name.
+        self.cmd('az connectedvmware inventory-item show_command -g {rg} --vcenter-name {vc_name} --inventory-item {rp_morefid}', checks=[
```

Comment:
`az connectedvmware inventory-item list` should also be tested

Reply:
This command is giving very long output that we are not fitting into the variable and getting out of buffer error, testing show and list are same, with list we will get all the output, with show we will get specific inventory item id.

Reply:
Will users get ‘out of buffer error’ if they use the 'show' command?

Reply:
'show' command output is finding because we are listing a particular inventory id so output length would be ridiculously small. In case of list, we are querying for all the inventory item, this is giving very hug output that not able to fit into the buffer.